edb/server/protocol/binary.pyx

```diff
@@ -1198,14 +1198,14 @@ cdef class EdgeConnection:
 
     cdef char render_cardinality(self, query_unit) except -1:
         if query_unit.cardinality is CARD_NO_RESULT:
-            return <char>(b'n')
-        elif query_unit.cardinality is CARD_ONE:
-            return <char>(b'o')
+            return <char>(CARD_NO_RESULT.value)
```

Comment:
I don't think there's a reason to explicitly filter for enum values here.  Just return `<char>query_unit.cardinality.value` directly.

Reply:
changed